src/app/body/task-details/upload-files/upload-files.component.html

```diff
@@ -17,7 +17,7 @@ <h6 class="col-md-11 col-10 pt-2 mb-0">Upload Files</h6>
         <div class="col-md-1 col-2 pt-1">
             <form>
                 <div class="form-group">
-                    <label class="btn" for="exampleFormControlFile1"><span class="material-icons uploadIcon">cloud_upload</span></label>
+                    <span class="material-icons uploadIcon">cloud_upload</span>
```

Comment:
Better use mat-icon wala code. Check in header component for reference

Reply:
done